install_proto.sh

```diff
@@ -0,0 +1,33 @@
+PROTOC_EXISTS=$?
```

Comment:
for my understanding, how is this populated? It is echoing the result of the prior command if I understand correctly, which in the context of the makefile is the dependency install -- is that correct?

Reply:
My understanding is `$?` with `_EXISTS` appended will return 0 if the beginning command is valid. So `PROTOC_EXISTS` checks `PROTOC` is installed `PROTO_EXISTS` checks that `PROTO` is installed.

Reply:
Though, that didn't seem to work with the github action, so I have updated.